### ISSUE TITLE
NPM package

### ISSUE THREAD

**Issue by imMatheus** [Eronred/handy-arrows#8]
Just wondering if there are any plans on making this an NPM package? I would love to help out if you think you would want it :)

PS, really love the SVGs! 💯

**Comment by Eronred** [comment]
Thank you for the idea and for using it! I'm glad to hear that it has been helpful!

IMO, using an illustration package is not ideal, as it can significantly increase the bundle size. It's more efficient to create a separate component for this purpose. With nearly 500 SVGs (more coming), the package size could be quite large. Thats why not sure about it.

The best approach is to clarify your use cases and create separate components for them. A tip is that you can animate these components with [Framer Motion](https://motion.dev/docs/react-quick-start), which is on the roadmap!

**Comment by imMatheus** [comment]
Yeah makes sense, bundle size would make this pretty hefty and not ideal
